### ISSUE TITLE
ratio change problem

### ISSUE THREAD

**Issue by Kelvin-hzm** [hsiangyuzhao/RCPS#19]
Hello, I'm new in medical image segmentation and I wanna change ratio in la.cfg. So I wonder what cps_ratio and con_ratio mean in la.cfg? Or I just need change the ratio into 0.2?

**Comment by hsiangyuzhao** [comment]
Hi @Kelvin-hzm 

`cps_ratio` denotes the ratio for rectified pseudo-supervision loss in the paper, while `con_ratio` denotes the ratio for bidirectional voxel contrastive loss in the paper. Please read our paper for further details.

Typically, the ratio for LA and Pancreas-CT task has been determined for best performance. However, you may explore different ratio settings for other tasks.

**Comment by Kelvin-hzm** [comment]
Thanks! By the way,  when I evaluate the train result, not only I saw the la_dice and so on, but I saw some result in bg_  as prefix. What does their mean?

**Comment by Kelvin-hzm** [comment]
> Hi @Kelvin-hzm
> 
> `cps_ratio` denotes the ratio for rectified pseudo-supervision loss in the paper, while `con_ratio` denotes the ratio for bidirectional voxel contrastive loss in the paper. Please read our paper for further details.
> 
> Typically, the ratio for LA and Pancreas-CT task has been determined for best performance. However, you may explore different ratio settings for other tasks.

Sorry for bothering you again, I see your paper and I try  to figure out how to change the label ratio(as your paper mentioned 2 results in 10% and 20% labeled data)?

**Comment by Kelvin-hzm** [comment]
Is it RATIO in la.cfg? So I guess I don't need change cps_ratio and con_ratio, right?

**Comment by hsiangyuzhao** [comment]
> not only I saw the la_dice and so on, but I saw some result in bg_ as prefix. What does their mean?

This means the dice of background class, as we use softmax rather than sigmoid for activation.

> I see your paper and I try to figure out how to change the label ratio(as your paper mentioned 2 results in 10% and 20% labeled data)?
> Is it RATIO in la.cfg? So I guess I don't need change cps_ratio and con_ratio, right?

That's right. You may change this argument for different ratio of labeled images.

**Comment by Kelvin-hzm** [comment]
> > not only I saw the la_dice and so on, but I saw some result in bg_ as prefix. What does their mean?
> 
> This means the dice of background class, as we use softmax rather than sigmoid for activation.
> 
> > I see your paper and I try to figure out how to change the label ratio(as your paper mentioned 2 results in 10% and 20% labeled data)?
> > Is it RATIO in la.cfg? So I guess I don't need change cps_ratio and con_ratio, right?
> 
> That's right. You may change this argument for different ratio of labeled images.

Thank you again for your patience! I got it.
